packages/common/src/proto-utils.ts

```diff
@@ -0,0 +1,103 @@
+import proto from '@temporalio/proto';
+import { fromProto3JSON, toProto3JSON } from 'proto3-json-serializer';
+import { patchProtobufRoot } from '@temporalio/proto/lib/patch-protobuf-root';
+
+export type History = proto.temporal.api.history.v1.IHistory;
+export type Payload = proto.temporal.api.common.v1.IPayload;
+
+// Cast to any because the generated proto module types are missing the lookupType method
+const patched = patchProtobufRoot(proto) as any;
```

Comment:
Safe to assume `patchProtobufRoot` can be re-executed on already-patched roots safely?